hail/build.sc

```diff
@@ -76,6 +76,10 @@ object Deps {
   }
 
   object Breeze {
+    // WARNING WARNING WARNING
+    // Before changing the breeze version review:
+    // - https://hail.zulipchat.com/#narrow/stream/123011-Hail-Query-Dev/topic/new.20spark.20ndarray.20failures/near/41645
+    // - https://github.com/hail-is/hail/pull/11555
     val core = ivy"org.scalanlp::breeze:1.1"
```

Comment:
It appears we've actually been using breeze 1.2, because spark-mllib pulls it in:
```
❯ mill ivyDepsTree --withCompile --withRuntime --whatDependsOn org.scalanlp:breeze_2.12                                                        (base)
[17/17] ivyDepsTree
└─ org.scalanlp:breeze_2.12:1.2
   ├─ org.apache.spark:spark-mllib-local_2.12:3.3.0
   │  ├─ org.apache.spark:spark-graphx_2.12:3.3.0
   │  │  └─ org.apache.spark:spark-mllib_2.12:3.3.0
   │  └─ org.apache.spark:spark-mllib_2.12:3.3.0
   ├─ org.apache.spark:spark-mllib_2.12:3.3.0
   └─ org.scalanlp:breeze-natives_2.12:1.1 org.scalanlp:breeze_2.12:1.1 -> 1.2
```

Reply:
Does the compileIvyDeps line not exclude them? I want to let Spark use their thing and relocate breeze for our own internal purposes.

Reply:
Ah, I missed that, that does let us use breeze 1.1. And I now realize even without that, because this is a compile only dependency, we would build with breeze 1.2 but run with 1.1.